spec/acceptance/requirements_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
+  context 'with default behaviour' do
+    let(:pp) do
+      'include posix_acl::requirements'
+    end
+
+    it 'works idempotently with no errors' do
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
```

Comment:
Shorter version using https://github.com/voxpupuli/voxpupuli-acceptance/#an-idempotent-resource
```suggestion
    it_behaves_like 'an idempotent resource' do
      let(:manifest) do
        'include posix_acl::requirements'
      end
    end
```